README.md

```diff
@@ -27,8 +27,11 @@ No installation required. Just include  "hashinator.h" . However, if you plan
 
 ## Run the tests   
 ```
-meson wrap install gtest # skip if you have gtest installed system wide and pkg-config can find it. 
-meson setup build  --buildtype=release 
+cd hashinator   
+mkdir subprojects
+meson wrap install gtest 
+meson setup build  --buildtype=release
+meson compile -C build --jobs=8
 meson test -C build
 ```
```

Comment:
README is still lacking instructions on how to use allocators

Reply:
This is now in the README